systemd/systemd.go

```diff
@@ -544,7 +570,12 @@ func (s *systemd) Status(unitNames ...string) ([]*UnitStatus, error) {
 }
 
 func (s *systemd) IsEnabled(serviceName string) (bool, error) {
-	_, err := s.systemctl("--root", s.rootDir, "is-enabled", serviceName)
+	var err error
+	if s.rootDir != "" {
+		_, err = s.systemctl("--root", s.rootDir, "is-enabled", serviceName)
+	} else {
+		_, err = s.systemctl("is-enabled", serviceName)
+	}
 	if err == nil {
 		return true, nil
 	}
```

Comment:
IsActive has also "--root", does it need to be changed? simply be dropped there always?

Reply:
Right, added same logic for consistency.